build.gradle

```diff
@@ -20,7 +20,7 @@ sourceCompatibility = 1.14
 targetCompatibility = 1.14
 
 allprojects {
-    version = '0.1.7'
+    version = '0.1.8'
```

Comment:
should this be a bump of minor version instead of patch since there is new functionality ?